src/pages/SetPasswordPage.js

```diff
@@ -96,7 +97,9 @@ class SetPasswordPage extends Component {
     render() {
         const buttonText = this.props.translate('setPasswordPage.setPassword');
         const sessionError = this.props.session.errors && ErrorUtils.getLatestErrorMessage(this.props.session);
-        const error = sessionError || ErrorUtils.getLatestErrorMessage(this.props.account);
+        const accountError = this.props.account.errors && ErrorUtils.getLatestErrorMessage(this.props.account);
+        const error = accountError || sessionError;
```

Comment:
This seems unnecessary. I think we should remove this, or rename it to something like `isSignInError`

Reply:
These are error strings, so naming with `is..` feels little bit weird
And I'm going to bring this up soon to move all relevant errors to one Onyx key and not keep in two different keys.

Reply:
Ah okay, that makes sense now 👍